src/components/tile-details-panel/normals.tsx

```diff
@@ -26,12 +29,42 @@ const NormalsContainer = styled.div`
 `;
 
 const NORMALS_ABSENCE_MESSAGE = "The tile has no normals";
+const NORMALS_PERCENTAGE_TITLE = "Percent of triangles with normals, %";
+const NORMALS_LENGTH_TITLE = "Normals length, m";
+
+type NormalsProps = {
+  tile: Tile3D;
+  trianglesPercentage: number;
+  normalsLength: number;
+  handleShowNormals: (tile: Tile3D) => void;
+  handleChangeTrianglesPercentage: (tile: Tile3D, percentage: number) => void;
+  handleChangeNormalsLength: (tile: Tile3D, length: number) => void;
```

Comment:
Callback name should be `onMakeSomething`. Handler might be `hanleMakeSomething` of `onMakeSomethingHandler`